config/config.txt

```diff
@@ -371,9 +374,9 @@ ANNOUNCE_ADMIN_LOGOUT
 MAPROTATION
 
 ## Map voting
-## Allows players to vote for their preffered map
+## Allows players to vote with their preffered map setting
 ## When it's set to zero, the map will be randomly picked each round
-ALLOW_MAP_VOTING 1
+PASSIVE_MAP_VOTING 1
```

Comment:
PREFERENCE_MAP_VOTING might be better ? Up to you.

Reply:
yeah, that sounds better